### ISSUE TITLE
FineTuning Model

### ISSUE THREAD

**Issue by steven0704** [ericsujw/InstColorization#16]
First of all, great job on the project! I was wondering if there were any implemented functionality to finetune this model? Basically introducing another dataset to train on top of the already trained model.

Thanks in advance.

**Comment by steven0704** [comment]
Hi, I am trying to train the model on my own dataset. 
After training is complete, there is no colourisation happening whatsoever. Is anyone able to train this model well and would give me some insight? Such as amount of epochs, loss and dataset size?
Cheers!

**Comment by delldu** [comment]
Hi Dear friend,
    If you use CPU test, not GPU, you will get gray result. Just set gpu_ids = "-1" in base_options.py. Do you test on CPU ? @steven0704
